### PR TITLE
Initializes Flatpak builder

### LINKED ISSUE #75: Produce Flatpak package on CI
So Linux users don't need to build from sources.

### PR DESCRIPTION
**Pull request by ultimaweapon**
Closes #75.

### CODE REVIEW

File: .github/workflows/ci-linux.yml [hunk 1]
```diff
@@ -8,29 +8,22 @@ jobs:
     steps:
     - name: Checkout source
       uses: actions/checkout@v3
-    - name: Install system dependencies
+    - name: Install System Packages
       run: |
-        wget -qO - https://packages.lunarg.com/lunarg-signing-key-pub.asc | sudo tee /etc/apt/trusted.gpg.d/lunarg.asc
```
Comment (yisonPylkita) [inline .github/workflows/ci-linux.yml:13]: Do you want to skip Vulkan SDK installation for now?

Comment (ultimaweapon) [reply .github/workflows/ci-linux.yml:13]: Yeah because we did not use it yet and it seems like we do not need it soon.

Comment (yisonPylkita) [reply .github/workflows/ci-linux.yml:13]: That's fine. These steps will stay in Git history anyway

Comment (ultimaweapon) [reply .github/workflows/ci-linux.yml:13]: Right. When we need it we can grab this from the history without required to keep maintenance it while we are not using.

**Review by bayedieng — APPROVED** [review]

**Pull request merged**